server.js

```diff
@@ -578,4 +579,113 @@ app.get('/graph', function (req, res) {
       });
   });
 
+/**
+ * Get account info
+ */
+app.get('/account/:account', function (req, res){
+    let req_account = req.params.account;
+    axios.all([
+        axios.get(SGAPI + '/account/' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/account/' + req_account + '/rewards', { httpsAgent: agent }),
+        axios.get(SGAPI + '/delegations?delegatorAddress=' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/transactions?&sender=' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/transactions?offset=20&sender=' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/transactions?offset=40&sender=' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/transactions?offset=60&sender=' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/transactions?offset=80&sender=' + req_account, { httpsAgent: agent }),
+        axios.get(CGAPI + '/simple/price?ids=cosmos&vs_currencies=usd', { httpsAgent: agent }),
+    ])
+        .then(axios.spread(function (data1, data2, data3, data4, data5, data6, data7, data8, data9) {
+            let available = 0;
+            if (data1.data.account.balance.length > 0)
+            {
+                available = parseInt(data1.data.account.balance[0].amount);
+            }
+            let delegated = 0;
+            if (data1.data.account.vested)
+            {
+                delegated = parseInt(data1.data.account.vestingInfo.delegatedFree[0].amount);
+            }
+            let rewards = 0;
+            if (data2.data.rewards.length > 0)
+            {
+                rewards = parseInt(data2.data.rewards[0].amount)
+            }
+            let transactions = [];
+            if (data4.data.transactions.length > 0) {
+                for (let i = 0; i < data4.data.transactions.length; i++)

```

Comment:
for loops can also be done with .forEach() which I find much easier to read and more concise to work with.

For instance, this one could be:
```
transactionsData.forEach(transaction => {
  transactions.push(transaction)
});
```